crates/ra_syntax/tests/data/lexer/0004_numbers.txt

```diff
@@ -47,9 +47,7 @@ IDENT 3 "foo"
 L_PAREN 1 "("
 R_PAREN 1 ")"
 WHITESPACE 1 "\n"
-INT_NUMBER 2 "0e"
-PLUS 1 "+"
-INT_NUMBER 1 "1"
+FLOAT_NUMBER 4 "0e+1"
```

Comment:
:+1:

Reply:
yeah, I was surprised that our old lexer had a bug :D